flocker_bb/builders/flocker.py

```diff
@@ -348,36 +338,36 @@ def makeInternalDocsFactory():
             '--verbose',
             '--delete-removed',
             '--no-preserve',
-            Interpolate('%s/%s/docs/' % (branch, revision)),
+            resultPath('docs'),
             Interpolate(
                 "s3://%(kw:bucket)s/%(prop:version)s/",
                 bucket='clusterhq-dev-docs',
             ),
         ],
-        path="private_html",
         doStepIf=isReleaseBranch('flocker'),
     ))
     return factory
 
 
-def createRepository(distribution):
+def createRepository(distribution, repository_path):
     steps = []
     flavour, version = distribution.split('-', 1)
     if flavour in ("fedora", "centos"):
-        steps.append(ShellCommand(
+        steps.append(MasterShellCommand(
```

Comment:
This where we build a repo containing multiple package versions, right?
Cool! It addresses a point I made in https://clusterhq.atlassian.net/browse/FLOC-1280
Will the master repo ever get cleaned up though? Or will it grow and grow?